.github/workflows/build.yaml

```
@@ -153,32 +153,33 @@ jobs:
           cd tests/containers/clients
           echo @opentdf:registry=https://npm.pkg.github.com >.npmrc
           echo //npm.pkg.github.com/:_authToken=${{ secrets.READ_PACKAGES_PAT }} >>.npmrc
-      - name: Run integration test Tiltfile
+      - name: Run xtest.Tiltfile
+        run: |-
+          export TILTFILE_FILE=xtest.Tiltfile
+          echo "TILTFILE_FILE=$TILTFILE_FILE" >> $GITHUB_ENV
+          tilt ci -f ${TILTFILE_FILE}
+          # Debugging prints
+          kubectl get deployments
+          kubectl get pods
```

```suggestion
          # Debugging prints
          kubectl get deployments
          kubectl get pods
```

We can _probably_ nuke this now that the "failure" catchall will fire if anything goes wrong.